models/borrowing.model.js

```diff
@@ -0,0 +1,22 @@
+const { DataTypes } = require("sequelize");
+
+module.exports = (sequelize) => {
+  const Borrowing = sequelize.define("borrowings", {
+    id: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+    },
+    updatedBy: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
+    returned_date: {
```

Comment:
Return date should not be nullable. They can borrow book indefinitely since it is not a charity